.circleci/config.yml

```diff
@@ -277,6 +281,7 @@ jobs:
       - run:
           name: Merge coverage files
           command: |
+            apt-get install -y curl
```

Comment:
Is this one necessary?

Reply:
Unbelievably, yes. Codecov uses curl to upload the report, that’s why it kept failing.

Reply:
Weird, but good to know.